sample/src/main/java/com/pedrogomez/renderers/sample/ui/diffing/VideoItemDiffCallback.java

```diff
@@ -0,0 +1,18 @@
+package com.pedrogomez.renderers.sample.ui.diffing;
+
+import android.support.annotation.NonNull;
+import android.support.v7.util.DiffUtil;
+import com.pedrogomez.renderers.sample.model.Video;
+
+public class VideoItemDiffCallback extends DiffUtil.ItemCallback<Video> {
```

Comment:
Couldn't we provide a default implementation based on the equality of the instances so the user doesn't need to implement this ``DiffCallback`` by default? I think we already did this with the ``diffUpdate`` implementation the ``RVRendererAdaper`` has. The user should be able to override this behavior, but by default we could provide this implementation based on a generic T instead.

Reply:
If we are able to add this default implementation as the last parameter we could easily replace the RVRendererAdapter with RVListRendererAdapter by just changing the constructor.

Reply:
Yes, we can! Then, do you think that the new adapter implementation should replace the older one (as it's going to keep the same signature on constructors) or that both implementations should be kept? 
I have added the changes with the default items' diffing implementation keeping both adapters, but we can change if you consider we should only have one.

Reply:
🤔I think we can keep it as is right now for the next minor release and I'll review if we can merge them soon.